subgraph.yaml

```diff
@@ -15,6 +15,11 @@ dataSources:
       language: wasm/assemblyscript
       entities:
         - DomainCreated
+        - Transfer
```

Comment:
These are supposed to correlate with the `schema.graphql` file
only have entities here which are entities in the graphql file.

Reply:
Remove
 - Transfer
        - MetadataLocked
        - MetadataUnlocked
        - MetadataChanged
       - RoyaltiesAmountChanged
       
       to fix